lib/lockstep_sdk/models/data_models.rb

```diff
@@ -21,110 +21,110 @@ module LockstepSdk
-        attr_accessor :modified              
+        attr_accessor :modified
         # @return [uuid] The ID of the user who last modified this activity.
-        attr_accessor :modifiedUserId        
+        attr_accessor :modified_user_id
         # @return [double] Amount collected (if any) for the activity.
-        attr_accessor :amountCollected       
+        attr_accessor :amount_collected
         # @return [double] Amount paid (if any) for the activity.
-        attr_accessor :amountPaid            
+        attr_accessor :amount_paid
         # @return [double] Credit given (if any) for the activity.
-        attr_accessor :creditGiven           
+        attr_accessor :credit_given
         # @return [boolean] True if this activity is to be shown in an "unread" state. When an activity is read by a person it is assigned to, this flag is set to false.
-        attr_accessor :isUnread              
+        attr_accessor :is_unread
         # @return [boolean] Activities may be archived when they should be hidden from the user. When this flag is true, this activity should be hidden.
-        attr_accessor :isArchived            
+        attr_accessor :is_archived
         # @return [company_model] The company associated with the activity To retrieve this collection, specify `Company` in the "Include" parameter for your query.
-        attr_accessor :company               
+        attr_accessor :company
         # @return [string] The name of the user the activity is assigned to
-        attr_accessor :userAssignedToName    
+        attr_accessor :user_assigned_to_name
         # @return [attachment_model] All attachments attached to applied activity. To retrieve this collection, specify `Attachments` in the "Include" parameter for your query.
-        attr_accessor :attachments           
+        attr_accessor :attachments
         # @return [note_model] All notes attached to this applied activity. To retrieve this collection, specify `Notes` in the "Include" parameter for your query.
-        attr_accessor :notes                 
+        attr_accessor :notes
         # @return [custom_field_definition_model] All definitions attached to this applied activity. To retrieve this collection, specify `CustomFieldValues` in the "Include" parameter for your query.
-        attr_accessor :customFieldDefinitions
+        attr_accessor :custom_field_definitions
         # @return [custom_field_value_model] All values attached to this activity. To retrieve this collection, specify `CustomFieldValues` in the "Include" parameter for your query.
-        attr_accessor :customFieldValues     
+        attr_accessor :custom_field_values
         # @return [activity_xref_model] All references attached to this applied activity. To retrieve this collection, specify `References` in the "Include" parameter for your query.
-        attr_accessor :references            
+        attr_accessor :references
     )
```

Comment:
this ending brackets would be replaced with the keyword `end`

Reply:
Fixed, thanks!